src/modules/login/views/ViewLogin.vue

```diff
@@ -0,0 +1,160 @@
+<script>
+import validateEmail from "@/utils/validateEmail";
+import validatePassword from "@/utils/validatePassword.js";
+import BaseInput from "@/components/BaseInput.vue";
+import BaseAlert from "@/components/BaseAlert.vue";
+import { loginUser } from "../services";
+import axios from "axios";
+export default {
+  data() {
+    return {
+      email: {
+        value: "",
+        error: "",
+      },
+      password: "",
+      password: {
+        value: "",
+        error: "",
+        seen: false,
+      },
+      submission: {
+        message: "",
+        isVerified: false,
+      },
+    };
+  },
+  components: {
+    BaseInput,
+    BaseAlert,
+  },
+  async created() {
+    try {
+      const response = await axios.get(`http://localhost:3000/login_users`);
```

Comment:
Why localhost?

Reply:
update: ;ocalhost has been changed to a live hosted testing server